### PR TITLE
Implementing the sub-sections of Help page.

### PR DESCRIPTION
**Pull request by developingright**
Hi @razzeee,
I have implemented the 'License' section of Help page. I just wanted to make sure if it's fine or some documentation tool or simple markdown would be better to use instead?

### CODE REVIEW

**Comment by razzeee** [comment]
While that would be fine in theory, this probably comes down to the question, if we want this page to be translated. If yes, it will make more sense to go for the markdown solution, as we want one translated text.

**Comment by developingright** [comment]
Hi @razzeee ,
I think it makes sense in the long run that we need a markdown solution for this page. I've been looking through several modules and came across the [elm-markdown ](https://package.elm-lang.org/packages/dillonkearns/elm-markdown/latest/)package. With it we can  define our own custom renderer, turning our markdown content into any data type with totally customizable logic and it’s markdown blocks are totally customizable.

**Comment by razzeee** [comment]
Feel free to use that. But I kinda expect https://package.elm-lang.org/packages/elm-explorations/markdown/latest/ might be easier for our usecase.

**Comment by developingright** [comment]
Hi @razzeee,
	I have implemented the License section in Help page using the [markdown package](https://package.elm-lang.org/packages/elm-explorations/markdown/latest/Markdown) which you suggested and have styled it in order to match Chorus2.

**Chorus 2 - License section** 
Screenshot from 2022-04-28 18-30-46

**Elm Chorus - License section**
Screenshot from 2022-04-28 18-30-34

File: public/index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <!-- CSS goes here -->
     <link rel="stylesheet" type="text/css" href="./main.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,600;1,700;1,800&display=swap" rel="stylesheet">
```
Comment (razzeee) [inline public/index.html:8]: We'll need to use a local version of the font, as we want to be able to use chorus when offline

File: public/main.css [hunk 2]
```diff
@@ -192,3 +192,28 @@
 .input-range-label:last-child {
   text-align: right;
 }
+.markdown{
+  font-family: 'Open Sans', sans-serif;
+}
+.markdown h1{
+  font-weight: 300;
+  font-size: 2.7em;
+
```
Comment (razzeee) [inline public/main.css:201]: Please format this with prettier

File: src/Pages/Help/License.elm [hunk 3]
```diff
@@ -58,16 +62,71 @@ subscriptions model =
...
+Included Libraries
+---------
+[Click here] to view the included libraries used by Chorus and their associated licenses.
+
+[Click here]: https://github.com/xbmc/chorus2/tree/master/src/lib
+[Jeremy Graham]: http://jez.me/
+[Pexels]: https://www.pexels.com/
+[Unsplash]: https://unsplash.com/
+[CC0 License]: https://unsplash.com/license
+[ CC0 License]: https://www.pexels.com/license/
```
Comment (razzeee) [inline src/Pages/Help/License.elm:110]: Is that space a typo?

File: src/Pages/Help/License.elm [hunk 4]
```diff
@@ -58,16 +62,71 @@ subscriptions model =
...
+[Unsplash - CC0 License]: https://unsplash.com/license
+[Creative Commons Zero]: https://creativecommons.org/publicdomain/zero/1.0/
+[along with this program;]: https://github.com/xbmc/chorus2/blob/master/LICENSE
```
Comment (razzeee) [inline src/Pages/Help/License.elm:113]: Is the `;` supposed to be there?

**Comment by developingright** [comment]
@razzeee,
I have formatted the code and added local font.

**Comment by razzeee** [comment]
Great work, do you still consider this work in progress?

**Comment by developingright** [comment]
Hi @razzeee ,
Apologies for the late response, as I've been traveling and was tied up with personal matter.

I'll remove the W.I.P status from this PR. The help page needs other section which is yet to be implemented. I'll add it in subsequent commit and create a PR so that the current changes are merged. 

Since this is a trivial task, I'll complete it in a short amount of time. Also since this task is mentioned in proposal (which will be done beforehand now), I'm planning to use some time to compare different open source media player softwares & try to adapt any new functionality/features and UI improvements that could add value to Elm Chorus.

**Pull request merged**